runtime/manual/node/migrate.md

```diff
@@ -7,91 +7,22 @@ oldUrl:
   - /runtime/manual/node/package_json
 ---
 
-One of Deno's core strengths is a unified toolchain that comes with support for
-TypeScript out of the box, and tools like a linter, formatter and a test runner.
-Switching to Deno allows you to simplify your toolchain and reduces the number
-of moving components in your project. Deno also has a more secure runtime, with
-[runtime permissions](../basics/permissions.md) that allow you to control what
-your code can access.
+Migrating a Node.js project to Deno is a straightforward process. Modern Node.js
+projects run out of the box with Deno with no changes to your code. The main
+points to be aware of are:
```

Comment:
Lead with "Deno supports modern Node projects out of the box." 

Had the user stops reading on the first sentence they're takeaway would be "Deno requires migrating" - that's what I want to work against. Always start with "Deno supports Node projects" in some form - then get into the details. 

It wouldn't hurt to restate 'Deno supports Node projects out of the box" over and over throughout the document.